src/components/app/router.ts

```diff
@@ -108,6 +107,16 @@ export const router = new Router([
     path:
       '/organisations/:organizationGUID/spaces/:spaceGUID/services/:serviceGUID/events/:eventGUID',
   },
+  {
+    action: services.listServiceLogs,
+    name: 'admin.organizations.spaces.services.logs.view',
+    path: '/organisations/:organizationGUID/spaces/:spaceGUID/services/:serviceGUID/logs',
+  },
+  {
+    action: services.downloadServiceLogs,
+    name: 'admin.organizations.spaces.services.logs.download',
+    path: '/organisations/:organizationGUID/spaces/:spaceGUID/services/:serviceGUID/logs/download',
```

Comment:
This could be:

```
/organisations/:organizationGUID/spaces/:spaceGUID/services/:serviceGUID/logs/:logFileName/download
```

which would allow us to build:

```
/organisations/:organizationGUID/spaces/:spaceGUID/services/:serviceGUID/logs/:logFileName/view
```

in the future. We can do this in a separate PR if we ever build view functionality.

Reply:
I'd agree. The problem is that the file names are: `errors/postgres.log-[timestamp]` which will likely mess up the url. Sadly that's the only "selectable" field allowed by AWS.